otlp/traces_direct_test.go

```diff
@@ -487,29 +492,6 @@ func TestUnmarshalTraceRequestDirect_Complete(t *testing.T) {
 		},
 	}
 
-	data := serializeTraceRequest(t, req)
```

Comment:
Note to reviewers, most of this code is the same, just moved; ignoring whitespace in the diff view should help.